src/components/buttons/Buttons.jsx

```diff
@@ -0,0 +1,14 @@
+import { MARK_ICON, PEN_ICON, BIN_ICON } from "src/utils/constants/constants";
+import { createButton } from "src/utils/helpers/createButton";
+
+const Buttons = () => {
+  return (
+    <div>
+      {createButton(MARK_ICON, "Mark task as done")}
```

Comment:
what is createButton representing? if it is component then represent it like below.
```suggestion
      <CreateButton markIcon={MARK_ICON} text="Mark task as done"/>
```
also use the text as const and import it.